tests/Unit/AI_Consent/User_Interface/AI_Consent_Integration/Enqueue_Assets_Test.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// phpcs:disable Yoast.NamingConventions.NamespaceName.TooLong -- Needed in the folder structure.
+// phpcs:disable Yoast.NamingConventions.NamespaceName.MaxExceeded
+namespace Yoast\WP\SEO\Tests\Unit\AI_Consent\User_Interface\AI_Consent_Integration;
+
+use Brain\Monkey;
+
+/**
+ * Tests the AI_Consent_Integration's enqueue_assets method.
+ *
+ * @group ai-consent
+ *
+ * @covers \Yoast\WP\SEO\AI_Consent\User_Interface\Ai_Consent_Integration::enqueue_assets
+ */
+final class Enqueue_Assets_Test extends Abstract_AI_Consent_Integration_Test {
+
+	/**
+	 * Tests enqueuing the assets.
+	 *
+	 * @covers ::enqueue_assets
```

Comment:
This creates a warning and is not needed since the cover annotation is done for the entire class.
```suggestion

```